wfe2/wfe_test.go

```diff
@@ -3356,7 +3356,6 @@ func TestPrepAuthzForDisplay(t *testing.T) {
 	wfe.prepAuthorizationForDisplay(&http.Request{Host: "localhost"}, authz)
 	chal := authz.Challenges[0]
 	test.AssertEquals(t, chal.URL, "http://localhost/acme/chall-v3/12345/po1V2w")
-	test.AssertEquals(t, chal.ProvidedKeyAuthorization, "")
```

Comment:
I think this is some of https://github.com/letsencrypt/boulder/pull/7655 leaking into this change?

Reply:
Good catch, reverted.